app/assets/stylesheets/openproject/_generic.sass

```diff
@@ -116,3 +116,6 @@
 
 .-no-z-index
   z-index: 0 !important
+
+.-error
+  color: $content-form-error-color
```

Comment:
Unfortunately, the color is overwritten by the resizer itself. Thus, there is no change in color right now. So please, add an `!important` here. Normally we try to avoid this. However, the are exceptions: One is for example a generic style class that shall definitely overwrite the normal element style (like in this case).

Reply:
It works on my machine (I see the resizer in red color when it reaches the min-width), so definitely I have to check what is happening on my local. I'll add the !important, thanks.